pkg/gits/git_cli.go

```diff
@@ -220,7 +220,12 @@ func (g *GitCLI) RemoteUpdate(dir string) error {
 
 // Stash stashes the current changes from the given directory
 func (g *GitCLI) Stash(dir string) error {
-	return g.gitCmd(dir, "stash")
+	return g.gitCmd(dir, "stash", "push")
+}
+
+// StashPop applies the last stash
+func (g *GitCLI) StashPop(dir string) error {
```

Comment:
What is the behaviour if there is no stash at the time `StashPop` is called?

I'd add some tests for `Stash` and `StashPop` to `TestGitCLI`